manager/src/test/java/com/redhat/service/bridge/manager/api/user/ProcessorAPITest.java

```diff
@@ -29,13 +28,7 @@
 import io.quarkus.test.junit.QuarkusTest;
 import io.restassured.response.Response;
 
-import static java.util.Arrays.asList;
-import static org.hamcrest.MatcherAssert.assertThat;
-import static org.hamcrest.Matchers.equalTo;
```

Comment:
Can you remove `org.hamcrest` from build-parent? Now it should have no remaining usage right?

Reply:
Hamcrest is currently not imported in any `pom.xml` file.
However, it's a dependency of `awaitility`, which uses Hamcrest `Matchers` in some APIs like, for example, to assert the content of the response body. These are currently used only in [End2EndTestIT.java](https://github.com/kostola/sandbox/blob/MGDOBR-40/runner/src/test/java/com/redhat/service/bridge/runner/it/End2EndTestIT.java).